ptp/protocol/management_tlvs.go

```diff
@@ -211,6 +228,14 @@ type ParentDataSetTLV struct {
 	GrandmasterIdentity                   ClockIdentity
 }
 
+// ClockAccuracyTLV
+type ClockAccuracyTLV struct {
+	ManagementTLVHead
+
+	ClockAccuracy uint8
```

Comment:
Can you use ClockAccuracy type which is already defined and based on uint8 anyway?